cmd/bridge/main.go

```diff
@@ -566,11 +569,11 @@ func main() {
 		apiServerEndpoint = srv.K8sProxyConfig.Endpoint.String()
 	}
 	srv.KubeAPIServerURL = apiServerEndpoint
-	srv.K8sClient = &http.Client{
-		Transport: &http.Transport{
-			TLSClientConfig: srv.K8sProxyConfig.TLSClientConfig,
-		},
+	k8sClient, err := server.GetK8sClient(srv.K8sProxyConfig.TLSClientConfig, bearerTokenFilePath)
+	if err != nil {
+		klog.Fatalf("Error inferring Kubernetes config from environment: %v", err)
 	}
+	srv.K8sClient = k8sClient
```

Comment:
don't you need to specifically call the `RoundTrip()` method in order to use the enhanced roundtripper functionality?